pkgs/build-support/kernel/make-initrd-ng/src/main.rs

```diff
@@ -12,10 +12,40 @@ use eyre::Context;
 use goblin::{elf::Elf, Object};
 use serde::{Deserialize};
 
+#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Copy, Debug, Deserialize, Hash)]
+#[serde(rename_all = "lowercase")]
+enum DLOpenPriority {
+    Required,
+    Recommended,
+    Suggested,
+}
+
+#[derive(PartialEq, Eq, Debug, Deserialize, Clone, Hash)]
+#[serde(rename_all = "camelCase")]
+struct DLOpenConfig {
+    use_priority: DLOpenPriority,
+    features: BTreeSet<String>,
+}
+
 #[derive(Deserialize, Debug)]
+struct DLOpenNote {
+    soname: Vec<String>,
+    feature: Option<String>,
+    // description: Option<String>,
```

Comment:
Remove?

Reply:
This is mainly a nod to the spec, which does include a description field that we don't need. I'd rather leave a comment explaining that than leave it out.